### ISSUE TITLE
frame drops and not compatible with other clients like obamahack 

### ISSUE THREAD

**Issue by spazheadd** [ionar2/salhack#284]
frame drops while flying 
fly around a spawn with godamor with more then 10 modules on
ion know like test
more then 10 modules and fly around 
bruh this sounds like a joke  but its a problem with salhack and salhack only ive tested with other clients

**Comment by Vaixe24Alt** [comment]
don't have access to these clients, please message @Vaixe24 on discord with certain modules or if you could give me a copy of obamahack to test out.

Many Thanks
Vaixe

**Comment by spazheadd** [comment]
> don't have access to these clients, please message @Vaixe24 on discord with certain modules or if you could give me a copy of obamahack to test out.
> 
> Many Thanks
> Vaixe

hi yeah i cant send on here as it does not accept jar files would i be able to send on discord ?

**Comment by spazheadd** [comment]
[ObamaHack.0.2.5 copy.jar.zip](https://github.com/ionar2/salhack/files/4762625/ObamaHack.0.2.5.copy.jar.zip)

**Comment by Vaixe24Alt** [comment]
Yeah, this is just a skid of another client, messy code as well, I believe that it is the other way round here. Also, seppuku is a not very compatible client.
